<launch>

    <arg name="sim_positions" default="$(find bwi_scavenger)/support/simulation.yaml" />
    <arg name="real_positions" default="$(find bwi_scavenger)/support/real.yaml" />

    <node name="scavenger" pkg="bwi_scavenger" type="scavenger" >
        <param name="path_to_search_points" value="$(arg real_positions)" />
    </node>

    <include file="$(find bwi_logging)/launch/record.launch" />

</launch>
